### DISCUSSION TITLE
I meet a puzzled problem when I calculate the IEA 15MW wind turbine

### DISCUSSION THREAD

**Discussion by haoshuning** [OpenFAST/openfast#2313] [General]
![2](https://github.com/OpenFAST/openfast/assets/160987005/418eb2ac-06af-42f7-9625-f4d7dd0485f1)
Hi, everyone
  When I calculate the motion response of IEA 15MW, the FAST encounter an error frequently, just as the picture shows. However it didn‘t show the specific error, I can't find the core problem, I really need your help.
Thank you sincerely!!!!!!!

**Comment by jjonkman** [comment]
Dear @haoshuning,

It looks like your model is numerically unstable.  Can you clarify what you changed in the OpenFAST input files relative to the OpenFAST model of the IEA Wind 15-MW reference wind turbine as provided in its repository (https://github.com/IEAWindTask37/IEA-15-240-RWT/tree/master/OpenFAST/IEA-15-240-RWT-Monopile)?

Best regards,

**Reply by haoshuning** [reply]
Dear jjonkman,
Thank you for your reply! Compared with the IEA Wind 15-MW reference wind turbine, I changed the arrangement of the mooring and the inflow wind document.My Moordyn and the inflow wind documents just as the picture show. 
![inflow](https://github.com/user-attachments/assets/9839f026-6676-4786-8d4f-40dc5c513820)
![MOORDYN](https://github.com/user-attachments/assets/d3bb6c65-55f0-422b-9067-b895ba368837)

**Reply by jjonkman** [reply]
Dear @haoshuning,

Can you isolate the issue to the mooring system change or the wind speed change (by changing one at a time)?

For the wind speed change, what is the mean wind speed you are expecting at hub height and are have you initialized the rotor speed and blade-pitch angles in ElastoDyn based on their expected (mean) values for that wind speed?  Otherwise, the controller may not recover and could cause the numerical instability.

Best regards,
